pkg/skaffold/kubernetes/colorpicker.go

```diff
@@ -53,11 +53,11 @@ type colorPicker struct {
 // sequentially from `colorCodes`. If all colors are used, the first color will be used
 // again. The formatter for the associated color will then be returned by `Pick` each
 // time it is called for the artifact and can be used to write to out in that color.
-func NewColorPicker(baseImageNames []string) ColorPicker {
+func NewColorPicker(imageNames []string) ColorPicker {
```

Comment:
It's unrelated to this change, but this colour-picking seems a bit busted.  The comment on `Pick` says it returns the colour associated to a pod, but in fact it is the colour associated to the first pod's container.  Which probably is a 1-1 association for most purposes, but it does mean that different pods with the same artifact may be coloured the same.